packages/settingeditor/src/SettingsFormEditor.tsx

```diff
@@ -189,6 +190,13 @@ export class SettingsFormEditor extends React.Component<
     this.setState({ isModified: false });
   };
 
+  private _syncFormDataWithSettings = (sender: any) => {
```

Comment:
Let's avoid `any` in new code (except for tests). I think this argument is not needed here:

```suggestion
  private _syncFormDataWithSettings = () => {
```